client/gutenberg/extensions/publicize/form.jsx

```diff
@@ -64,7 +63,6 @@ const PublicizeForm = compose( [
 		messageChange( event ) {
 			dispatch( 'core/editor' ).editPost( {
 				meta: {
-					...meta,
```

Comment:
Should we add a comment pointing to Gutenberg's code that does the merge for us to prove that we're not overwriting `meta` here?

Reply:
I think we're good without a comment.